cfn-resources/access-list-api-key/cmd/resource/resource.go

```diff
@@ -89,8 +90,8 @@ func Create(req handler.Request, prevModel *Model, currentModel *Model) (handler
 	}
 
 	// createReq.ApiService.
-	entryList := make([]atlasSDK.UserAccessList, 0)
-	var access atlasSDK.UserAccessList
+	entryList := make([]admin.UserAccessListRequest, 0)
+	var access admin.UserAccessListRequest
 	if currentModel.CidrBlock != nil {
```

Comment:
can CidrBlock and IPAddress exist at the same time?

Reply:
we are throwing an error [here](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/access-list-api-key/cmd/resource/resource.go#L84) in the create (and update is not supported in this resource).